plugins/scaffolder/src/alpha/components/TemplateEditorPage/TemplateEditorToolbar.tsx

```diff
@@ -120,32 +131,37 @@ export function TemplateEditorToolbar(props: {
           aria-labelledby="publish-dialog-title"
           aria-describedby="publish-dialog-description"
         >
-          <DialogTitle id="publish-dialog-title">Publish changes</DialogTitle>
+          <DialogTitle id="publish-dialog-title">
+            {t('templateEditorToolbar.addToCatalogDialogTitle')}
+          </DialogTitle>
           <DialogContent dividers>
             <DialogContentText id="publish-dialog-slide-description">
-              Follow the instructions below to create or update a template:
+              {t(
+                'templateEditorToolbar.addToCatalogDialogContent.introduction',
+              )}
               <ol>
-                <li>Save the template files in a local directory</li>
                 <li>
-                  Create a pull request to a new or existing git repository
+                  {t('templateEditorToolbar.addToCatalogDialogContent.step1')}
```

Comment:
Is it possible to have the separate step be newlines or smth within the translation messages instead? Feeling this a bit too wonky to ship

Reply:
Refactored to add a step for each new line.